lua/lvim/core/cmp.lua

```diff
@@ -109,9 +106,9 @@ local function jumpable(dir)
   end
 
   if dir == -1 then
-    return luasnip.in_snippet() and luasnip.jumpable(-1)
+    return luasnip.in_snippet() == true and luasnip.jumpable(-1)
```

Comment:
```suggestion
    return luasnip.in_snippet() and luasnip.jumpable(-1)
```

Reply:
Without `== true` it is null or true or false.

Based on the linting or type checking, this throws a warning for me which is why it was changed. Does this not cause a warning for you?

Reply:
oh I forgot to say that we should change the type annotation for the return value instead.

```lua
---@return boolean|nil true if a jumpable luasnip field is found while inside a snippet
```